topics/02_Controls/igGrid/03_Features/13_Virtualization/00_igGrid_Virtualization_Overview.md

```diff
@@ -10,74 +10,100 @@
 
 # Virtualization Overview (igGrid)
 
-## Topic Overview
 
-### Purpose
+## In this topic
 
-This topic introduces the Virtualization feature of the `igGrid`™ control.
+This topic contains the following sections:
 
+-   [**Introduction**](#introduction)
+-   [**Supported Virtualization Types**](#supported_virtualization_types)
+	-   [Fixed Row Virtualization](#fixed-row)
+	-   [Column Virtualization](#column)
+	-   [Continuous Row Virtualization](#continuous)
+-   [**Keyboard Interactions**](#keyboard-interactions)
+-   [**Related Content**](#related-content)
+    -   [Topics](#topics)
+	-   [Samples](#samples)
 
+## <a id="introduction"></a> Introduction
 
-## Introduction
+Virtualization is a feature of the `igGrid` that boosts performance when displaying large data sets containing thousands of records. It improves both rendering time and scrolling as well as memory footprint. It works by reducing the number of the DOM objects in-memory and reusing them while user scrolls and operates with the data. 
```

Comment:
The topic still doesn't state that this feature is not the feature of choice if you can enable, for example, paging. Also I didn't find anything related to the optimal number of records this feature is suitable for.